react/fields/Dropdown/Dropdown.test.js

```diff
@@ -93,6 +90,27 @@ describe('Dropdown', () => {
     });
   });
 
+  describe('Option Group', () => {
+    const opts = [{ label: 'suburbs', value: [{ label: 'truganina', value: '3029' }, { label: 'wl', value: '3029' }] }];
+    it('should render optgroup correctly', () => {
+      render(<Dropdown inputProps={{ value: '' }} options={opts} />);
```

Comment:
given all the tests are rendering the same thing, you can optimise this be calling `render` in a `before` within this `describe`. Then it'll just be firing off all the expectations nice and fast after a single render.